index.html

```diff
@@ -0,0 +1,850 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/16d2250538.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>TNW</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo-flex flex">
+          <img
+            class="top-logo sec-1-img"
+            src="images/toplogo.png"
+            alt="Pic here"
+          />
+          <div class="float-left">
+            <div class="nav-1 flex">
+              <div class="nav-1-left">
+                <a href="#">News</a>
+                <a href="#">Events</a>
+                <a href="#">Business</a>
+                <a href="#">AMAs</a>
+                <a href="#">Space</a>
+                <a href="#">Terms & Conditions</a>
+              </div>
+
+              <div class="nav-1-right">
+                <a href="#">About</a>
+                <a href="#">Advertise</a>
+                <a href="#">Jobs</a>
+                <a href="#">Contact</a>
+              </div>
+            </div>
+            <div class="nav-2-3">
+              <div class="nav-2 flex">
+                <div class="nav-2-left">
+                  <a href="#">LATEST</a>
+                  <a href="#">HARD FORK</a>
+                  <a href="#">PLUGGED</a>
+                  <a href="#">README</a>
+                  <a href="#">GROWTH QUARTER</a>
+                  <a href="#" class="no-disp">SHIFT</a>
+                  <a href="#" class="no-disp">NEURAL</a>
+                </div>
+                <div class="nav-2-right">
+                  <i class="fas fa-search nav-2-items"></i>
+                  <i class="fa fa-bars hamberger" aria-hidden="true"></i>
+
+                  <img
+                    class="nav-2-items no-disp"
+                    src="images/tnw2020.png"
+                    alt="TNW 2020"
+                  />
+                  <span class="no-disp">34 :22 :28 :50 </span>
+                  <button class="nav-2-btn nav-2-items no-disp" type="button">
+                    GET TICKETS
```

Comment:
####  Large Screen Size

- [ ]  Delete this whitespace between the navbar and the Jumbotron

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91842262-21519e00-ec2a-11ea-8c92-a6b0b13312b3.png) | ![image](https://user-images.githubusercontent.com/61479648/91842297-31697d80-ec2a-11ea-80b4-301822e25fe9.png) |

#### Medium Screen Size and Mobile Screen SIze

- [ ] The big TNW from the news articles should not show up 

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91842540-a2109a00-ec2a-11ea-8624-665b84389d98.png) |![image](https://user-images.githubusercontent.com/61479648/91842598-b6ed2d80-ec2a-11ea-8e05-42ec679d2a4e.png) |